app/api/admin/index.mjs

```diff
@@ -0,0 +1,15 @@
+import { getWebMentions } from '../../../shared/webmentions.mjs'
+
+/** @type {import('@enhance/types').EnhanceApiFn} */
+export async function get({ method, path, session }) {
+  console.log(`<${Date.now()}> ${method} ${path}`)
```

Comment:
do you think we should ditch the logs? it would lessen the amount of vars destructured from the params